src/Features/LanguageServer/ProtocolUnitTests/Options/LspOptionsTests.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.AddImport;
+using Microsoft.CodeAnalysis.CodeGeneration;
+using Microsoft.CodeAnalysis.CodeStyle;
+using Microsoft.CodeAnalysis.Editor.Test;
+using Microsoft.CodeAnalysis.Editor.UnitTests;
+using Microsoft.CodeAnalysis.Formatting;
+using Microsoft.CodeAnalysis.Options;
+using Microsoft.CodeAnalysis.Simplification;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.CodeAnalysis.LanguageServer.UnitTests.References;
+public class LspOptionsTests : AbstractLanguageServerProtocolTests
+{
+    public LspOptionsTests(ITestOutputHelper? testOutputHelper) : base(testOutputHelper)
+    {
+    }
+
+    protected override TestComposition Composition => EditorTestCompositions.LanguageServerProtocol
+        .AddAssemblies(Assembly.Load("Microsoft.CodeAnalysis.CSharp.LanguageServer.Protocol"))
```

Comment:
@tmat the references to these assemblies here and in the project should be removed when you remove the assembly, but the below tests still needs to pass.

I verified the tests fail without including these assemblies right now.